tests/tls.d

```diff
@@ -226,6 +227,24 @@ void testConn(TLSVersion cli_version, TLSVersion srv_version, bool expect_succes
 				stunnel.finalize();
 				return;
 			}
+			// test logging the keyfile
+			import vibe.stream.openssl;
+			static if (haveKeylog) {
+				if (auto osslctx = cast(OpenSSLContext)cctx) {
+					osslctx.keylogOnEnvVar();
+				}
```

Comment:
We shouldn't have to do this. It should be automatic if the variable is set.

Reply:
Given that we just realized the next version of openssl is going to have this behavior built in, do we really want to add this as default behavior now?

Reply:
done.